doc/manual/parallel-computing.rst

```diff
@@ -29,25 +29,12 @@ operations typically do not look like "message send" and "message
 receive" but rather resemble higher-level operations like calls to user
 functions.
 
-Parallel programming in Julia is built on two primitives: *remote
-references* and *remote calls*. A remote reference is an object that can
-be used from any process to refer to an object stored on a particular
-process. A remote call is a request by one process to call a certain
-function on certain arguments on another (possibly the same) process.
-
-Remote references come in two flavors -``Future`` and ``RemoteChannel``.
+.. _man-local-processes:
 
-A remote call returns a ``Future`` to its result. Remote calls
-return immediately; the process that made the call proceeds to its
-next operation while the remote call happens somewhere else. You can
-wait for a remote call to finish by calling :func:`wait` on the returned
-`Future`, and you can obtain the full value of the result using
-:func:`fetch`.
-
-On the other hand ``RemoteRefs`` are rewritable. For example, multiple processes
-can co-ordinate their processing by referencing the same remote ``Channel``\ .
+Adding Local Processes
+----------------------
 
-Let's try this out. Starting with ``julia -p n`` provides ``n`` worker
+Starting with ``julia -p n`` provides ``n`` worker
 processes on the local machine. Generally it makes sense for ``n`` to
 equal the number of CPU cores on the machine.
```

Comment:
+1, not sure I know the right answer though (or if it depends on details of the problem).